### PR TITLE
remove open ai

### PR DESCRIPTION
**Pull request by JnsCas**

### CODE REVIEW

File: src/application/controllers/language/LanguageController.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { FastifyReply, FastifyRequest } from 'fastify'
+import { createLogger } from '@quorum/elisma/src/infra/log'
+import { ProjectLanguage } from '@quorum/elisma/src/domain/bundle/entities/ProjectLanguage'
+
+const logger = createLogger('LanguageController')
+
+export class LanguageController {
+  async getAll(req: FastifyRequest, res: FastifyReply): Promise<void> {
+    logger.info(`Retrieving languages...`)
+    return res.send(Object.values(ProjectLanguage)) //FIXME get it from manifest
```
Comment (f-nyx) [inline src/application/controllers/language/LanguageController.ts:10]: There is an opened PR to add the languages to the manifests. You should use the new ManifestService to retrieve the list of manifests and put all languages in a Set.

File: src/application/controllers/library/LibraryController.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { FastifyReply, FastifyRequest } from 'fastify'
+import { createLogger } from '@quorum/elisma/src/infra/log'
+import { SupportedLibraries } from '@quorum/elisma/src/SupportedLibraries'
+
+const logger = createLogger('LibraryController')
+
+export class LibraryController {
+  getAll(req: FastifyRequest, res: FastifyReply) {
+    logger.info(`Retrieving all the libraries...`)
+    return res.send(SupportedLibraries)
```
Comment (f-nyx) [inline src/application/controllers/library/LibraryController.ts:10]: you should return the manifests here

**Review by f-nyx — APPROVED** [review]

File: src/application/controllers/language/LanguageController.ts (hunk 1 above)
Comment (JnsCas) [reply src/application/controllers/language/LanguageController.ts:10]: I'll do it in my next PR

File: src/application/controllers/library/LibraryController.ts (hunk 2 above)
Comment (JnsCas) [reply src/application/controllers/library/LibraryController.ts:10]: I'll do it in my next PR

**Pull request merged**